components/SecondHeading/index.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+const SecondHeading = styled.h2`
```

Comment:
would you like only to style h2? how about cancel the component and put it in pages/index?

Reply:
yes this is only for h2. You mean I should copy the code and put it in pages/index? Can you explain why?

Reply:
You could keep the file, but rename it into `SecondHeading.styled.js`: by doing so, every developer knows that this is only a styled component you can use wherever you need it.

Reply:
the same reason as Paul's. index.js should export a function. Before  I didn't notice that, maybe you would like to use it in other page. Then let it as a component, will be better.